tests/main.cpp

```
@@ -90,6 +117,29 @@ int main(int argc, const char** argv)
+                {
+                    if (str == rhi::testing::deviceTypeToString(deviceType))
+                    {
+                        options.deviceRequired[size_t(deviceType)] = true;
+                        matched = true;
+                        break;
+                    }
+                }
+                if (!matched)
+                {
+                    std::fprintf(stderr, "Invalid required device type '%s'.\n", str.c_str());
+                    return 1;
```

Cppcheck (2.21.0)

[style] The function 'setInternalReferenceCount' is never used.

(unusedFunction)